include/packager/live_packager_export.h

```diff
@@ -0,0 +1,95 @@
+//
+// Created by Oleksandr Khodos on 13.03.2024.
+//
+
+#ifndef PACKAGER_LIVE_PACKAGER_EXPORT_H
+#define PACKAGER_LIVE_PACKAGER_EXPORT_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef enum OutputFormat {
+    OUTPUT_FORMAT_FMP4,
+    OUTPUT_FORMAT_TS,
+    OUTPUT_FORMAT_VTTMP4,
+    OUTPUT_FORMAT_TTMLMP4,
+    OUTPUT_FORMAT_TTML,
+} OutputFormat_t;
+
+typedef enum TrackType {
+    TRACK_TYPE_AUDIO,
+    TRACK_TYPE_VIDEO,
+    TRACK_TYPE_TEXT,
+} TrackType_t;
+
+typedef enum EncryptionScheme {
+    ENCRYPTION_SCHEME_NONE,
+    ENCRYPTION_SCHEME_SAMPLE_AES,
+    ENCRYPTION_SCHEME_AES_128,
+    ENCRYPTION_SCHEME_CBCS,
+    ENCRYPTION_SCHEME_CENC,
+} EncryptionScheme_t;
+
+enum KeySize {
```

Comment:
There is no way to do that in C

Reply:
no constexpr!!

Reply:
The alternatives are `#define` and `static const ...`.

We actually don't need to name the enum, i.e.,
```
enum {
  IV_MAX_SIZE = 16,
  KEY_SIZE = 16,
  KEY_ID_SIZE = 16,
};
```

Reply:
You cannot use them to define fixed-size arrays. I wanted to make it explicit, instead of using shaka's approach to put "16" magic constant everywhere.
`static const` variables cannot be used as a size of a fixed-size array, although I would "see" such variables on Go side

Reply:
Defines are not visible on Go side, `static const` works fine

Reply:
Done

Reply:
Does not work on GCC
```
./live_packager_export.h:46:11: error: variably modified 'iv' at file scope
   46 |   uint8_t iv[kIvMaxSize];
      |           ^~
./live_packager_export.h:47:11: error: variably modified 'key' at file scope
   47 |   uint8_t key[kKeySize];
      |           ^~~
./live_packager_export.h:48:11: error: variably modified 'key_id' at file scope
   48 |   uint8_t key_id[kKeyIdSize];
      |           ^~~~~~
```

Reply:
Defines seem to work fine

Reply:
yes, for C defines is a way to go.